packages/hardhat/contracts/YourContract.sol

```diff
@@ -1,4 +1,5 @@
-pragma solidity >=0.6.0 <0.8.0;
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.7.0;
```

Comment:
If you pull main on your branch, I switch the solidity version to 0.8 in hardhat configs

Reply:
Look at this PR : https://github.com/1Hive/quests/pull/7